js/download.js

```diff
@@ -116,3 +116,17 @@ function drawImageAndQuote(img, ctx) {
     );
   }
 }
+
+
+function saveToDevice(){
+  if (window.navigator.msSaveBlob) {
+    window.navigator.msSaveBlob(canvas.msToBlob(), "Wallpaper.jpg");
```

Comment:
This is a Non-standard Deprecated Method
Why not just use the other one which inside the else?